extensions/Push/lib/Connector/Phabricator.pm

```diff
@@ -59,9 +54,32 @@ sub should_send {
 }
 
 sub send {
+    my ( $self, $message ) = @_;
+
     my $logger = Bugzilla->push_ext->logger;
 
-    $logger->info('AUDIT');
+    my $data = $message->payload_decoded;
+    my $bug_data = $self->_get_bug_data($data) || return 0;
+    my $bug = Bugzilla::Bug->new( { id => $bug_data->{id}, cache => 1 } );
+
+    my @synced_groups = split(/,/, Bugzilla->params->{phabricator_sync_groups});
+
+    if ( any { /$_->{name}/ ~~ @synced_groups } @{$bug->groups()} ) {
+        my @attachments = grep {
+            $_->isobsolete == 0 &&
+            $_->contenttype eq PHAB_CONTENT_TYPE &&
+            $_->attacher->{login_name} eq 'phab-bot@bmo.tld'
+        } @{ $bug->attachments() };
+
+        if(@attachments){
+            my @rev_ids;
+            foreach my $attachment (@attachments) {
+                my ($rev_id) = ($attachment->filename =~ PHAB_ATTACHMENT_PATTERN);
+                next if !$rev_id;
+                push(@rev_ids, $rev_id);
+            }
```

Comment:
The code to make the actual changes should exist now in extension/PhabBug/lib/Util.pm to make the changes to the revision ids found.